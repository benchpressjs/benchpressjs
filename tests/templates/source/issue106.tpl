{{{each users}}}
  <li>
    <a href="{config.relative_path}/uid/{../uid}">{{buildAvatar(users, "sm", true)}} {../username}</a>
  </li>
{{{end}}}
