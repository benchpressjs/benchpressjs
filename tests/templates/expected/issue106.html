<li>
  <a href="/uid/21591"><img class="avatar avatar-sm" alt="Noah R" title="Noah R" data-uid="21591" loading="lazy" component="avatar/picture" src="https://lh5.googleusercontent.com/-UUo7LvH6tsM/AAAAAAAAAAI/AAAAAAAAAAA/AMZuucmJGE-or0l_ziNT1zwYpuZcb5rVEA/s96-c/photo.jpg" style="" />Noah R</a>
</li>
<li>
  <a href="/uid/21590"><span class="avatar avatar-sm" alt="oriettapolo" title="oriettapolo" data-uid="21590" loading="lazy" component="avatar/icon" style="background-color: #795548;">O</span>oriettapolo</a>
</li>